src/main/java/org/folio/circulation/domain/representations/CheckOutByBarcodeRequest.java

```diff
@@ -7,4 +7,6 @@ private CheckOutByBarcodeRequest() { }
   public static final String USER_BARCODE = "userBarcode";
   public static final String PROXY_USER_BARCODE = "proxyUserBarcode";
   public static final String SERVICE_POINT_ID = "servicePointId";
+  public static final String DUE_DATE = "dueDate";
```

Comment:
The checkout by barcode request does not include these properties, only the override request.

I think it would be clearer to separate these out, in case it is believed they are included in a check out request.

Reply:
Done